examples/kitchen-sink.yaml

```diff
@@ -397,3 +397,6 @@ instrumentation/development:
   swift:
     example:
       property: "value"
+distribution:
+  example:
```

Comment:
is `example` the name of a distribution here? if not, should it be? e.g. in Java Instrumentation we have at least two (upstream) distributions `agent` and `spring_starter`, I could see it being helpful to know what kind of config yaml we're looking at

or if not, maybe `distribution` could have a (mandatory?) `name:`?